mozetl/search/daily_search_rollups.py

```diff
@@ -129,6 +130,7 @@ def roll_up_searches(spark, frame):
         "distribution_id",
         "default_provider",
         "search_counts.engine as search_provider",
+        "search_counts.source as search_source",
```

Comment:
See above. The bug links should be the most useful resource for future engineers.